### PR TITLE
[tw-1158] Refresh configuring team settings

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Added hero image, and updated outdated screenshots
* Added team role requirements for managing a team profile
* Added additional H2s for newer items added to the left sidebar in Team Settings: Secret Scanner, Installed apps, Invite links, and Enterprise application
    * Chose not to add new sections for Solutions access, Private API Network, and Set up API authorization -- these sections seem to focus on explaining a feature (or add-on) or apply to earlier versions
* Reorganized content as needed
* Ran Vale and found no major issues
* Removed deprecated key-values from front matter, and changed the last updated date

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 💎✨

File: src/pages/docs/administration/team-settings.md [hunk 1]
```diff
@@ -72,67 +76,85 @@ When you select **Team Settings**, you will be automatically directed to your **
...
 
-If you enable your team's public profile, your team will be visible on Postman's [API Network](/docs/collaborating-in-postman/adding-private-network/), along with any [APIs](/docs/collaborating-in-postman/adding-private-network/#adding-apis), [collections](/docs/publishing-your-api/publishing-your-docs/), and [workspaces](/docs/collaborating-in-postman/using-workspaces/public-workspaces/) your team has published.
+A public team profile encourages other users to collaborate with you in a public workspace, enabling you to ask for contributions, gather feedback, and increase your API's discovery in search results.
 
-A public team profile encourages other users to collaborate with you in a public workspace, enables you to ask for contributions and gather feedback, and increases your API's discovery in search results. A public team profile also shows up on the [Public API Network](https://www.postman.com/explore).
+If you enable your team's public profile, your team will be visible on [the Public API Network](/docs/collaborating-in-postman/public-api-network/public-api-network-overview/), along with your public workspaces, collections, APIs, and flows.
```
Comment (gitThingsDone) [inline src/pages/docs/administration/team-settings.md:95]: @MatthewPinsker-postman should we instead link to postman.com/explore ?

Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/team-settings.md]: Thanks for bringing this up, @gitThingsDone. Please see the latest commit for updates based on your feedback: https://github.com/postmanlabs/postman-docs/pull/4884/commits/c45948a09147f1854c30e634562d3c1c21feb374

**Review by gitThingsDone — APPROVED** [review]
LGTM

**Pull request merged**